core/main/src/service/user_grants.rs

```diff
@@ -620,6 +623,7 @@ impl GrantState {
             "Incoming grant check for app_id: {:?} and role: {:?}",
             app_id, role_info
         );
+        #[allow(clippy::unnecessary_fallible_conversions)]
```

Comment:
Can we use `from` instead of `try_from`?

Reply:
i could not get it to compile, but probably a skillz issue on my side, and i was also a little concerned about why it was try_from the 1st place- - the conversion is infallible from the looks of it, but did not want to assume and break stuff.

Reply:
I think this compiles. Maybe at one point role_info was a generic type or something hence the `try_from`?

`let permission = FireboltPermission::from(role_info);`